micromamba/tests/test_create.py

```diff
@@ -143,6 +143,23 @@ def test_env_lockfile_different_install_after_create(tmp_home, tmp_root_prefix,
     helpers.install("-p", env_prefix, "-f", install_spec_file, "-y", "--json")
 
 
+# Only run this test on Linux, as it is the only platform where xeus-cling
+# (which is part of the environment) is available.
+@pytest.mark.skipif(platform.system() != "Linux", reason="Test only available on Linux")
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+def test_env_spdlog_overhead_regression(tmp_home, tmp_root_prefix, tmp_path):
+    # Non-regression test
+
+    env_prefix = tmp_path / "myenv"
+    create_spec_file = tmp_path / "env-spdlog-callback.yaml"
+
+    shutil.copyfile(__this_dir__ / "env-spdlog-callback.yaml", create_spec_file)
+
+    # Must not hang and complete successfully
+    res = helpers.create("-p", env_prefix, "-f", create_spec_file, "-y", "--json")
+    assert res["success"]
```

Comment:
I think that this will succeed at the end, the issue (failure) should be if it takes an abnormal amount of time to finish. So maybe add an assert on the execution time of this test to not exceed a certain limit? (The limit value would be defined based on the local tests that you had for example).

Reply:
That's a good remark. I guess we could use `pytest-timeout` for this so that the test automatically fails after a given period. What do you think?